src/java.base/share/classes/java/lang/reflect/AccessFlag.java

```diff
@@ -352,19 +356,14 @@ public Set<Location> apply(ClassFileFormatVersion cffv) {
      * major versions 46 through 60, inclusive (JVMS {@jvms 4.6}),
      * corresponding to Java SE 1.2 through 16.
      */
-    STRICT(0x0000_0800, true, !ValhallaFeatures.isEnabled() ? Location.EMPTY_SET : Location.SET_FIELD,
+    STRICT(Modifier.STRICT, true, Location.EMPTY_SET,
              new Function<ClassFileFormatVersion, Set<Location>>() {
                @Override
                public Set<Location> apply(ClassFileFormatVersion cffv) {
-                   if (ValhallaFeatures.isEnabled() && cffv.compareTo(ClassFileFormatVersion.RELEASE_22) >= 0) {
-                       return Location.SET_FIELD;
-                   } else if (cffv.compareTo(ClassFileFormatVersion.RELEASE_2) >= 0 &&
-                              cffv.compareTo(ClassFileFormatVersion.RELEASE_16) <= 0) {
-                       return Location.SET_METHOD;
-                   } else {
-                       return Location.EMPTY_SET;
-                   }
-               }
+                   return (cffv.compareTo(ClassFileFormatVersion.RELEASE_2)  >= 0 &&
+                           cffv.compareTo(ClassFileFormatVersion.RELEASE_16) <= 0) ?
```

Comment:
I believe we still want to keep the strict field modifier for value class fields.

Reply:
I dropped it because its not in JEP401 and the feature has not settled yet nor which spec its covered in.

Reply:
I'm okay either way if you want to make change now or later.   If the testing is good, it's fine to integrate this change and follow up later.